src/protocols/secure_channel/PairingSession.h

```diff
@@ -174,6 +163,29 @@ class DLL_EXPORT PairingSession : public SessionDelegate
         else
         {
             err = OnFailureStatusReport(report.GetGeneralCode(), report.GetProtocolCode());
+
+            if (report.GetGeneralCode() == Protocols::SecureChannel::GeneralStatusCode::kBusy &&
+                report.GetProtocolCode() == Protocols::SecureChannel::kProtocolCodeBusy)
+            {
+                if (!report.GetProtocolData().IsNull())
+                {
+                    Encoding::LittleEndian::Reader reader(report.GetProtocolData()->Start(),
+                                                          report.GetProtocolData()->DataLength());
+
+                    // TODO: https://github.com/project-chip/connectedhomeip/issues/28290
+                    uint16_t minimumWaitTime = 0;
+                    err                      = reader.Read16(&minimumWaitTime).StatusCode();
```

Comment:
So what is done with the wait time? Logging is insufficient. If we don't care and don't use for now, let's not log at all, and just do unit tests for the fact that our code will encode it properly.

Reply:
There's a followup issue to propagate that time to the SDK consumer pairing session consumer.